src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/StandardOleMarshalObject.cs

```diff
@@ -8,26 +8,28 @@ namespace System.Runtime.InteropServices
 {
     public class StandardOleMarshalObject : MarshalByRefObject, IMarshal
     {
-        private static readonly Guid CLSID_StdMarshal = new Guid("00000017-0000-0000-c000-000000000046");
+        private static readonly Guid StdMarshalClsid = new Guid("00000017-0000-0000-c000-000000000046");
```

Comment:
Please don't change class ID variable constant name. The format matches the native format and it helps searching for them. I realize this is not corefx policy, but in this case COM rules win.

Reply:
> I realize this is not corefx policy

It is corefx naming conventions to keep interop related names as is: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md#naming

Reply:
Oh nice. Didn't realize that was spelled out already - Yay.

Reply:
Fixed